package.json

```diff
@@ -19,8 +19,8 @@
     "dist:mmi:debug": "yarn dist --build-type mmi --apply-lavamoat=false --snow=false",
     "build": "yarn lavamoat:build",
     "build:dev": "node development/build/index.js",
-    "start:test": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' SENTRY_DSN_DEV=https://fake@sentry.io/0000000 yarn build:dev testDev",
-    "start:test:flask": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' SENTRY_DSN_DEV=https://fake@sentry.io/0000000 yarn build:dev testDev --build-type flask --apply-lavamoat=false --snow=false",
+    "start:test": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' SENTRY_DSN_DEV=https://fake@sentry.io/0000000 yarn build:dev testDev --apply-lavamoat=false",
```

Comment:
does it mean we were running lavamoat in all tests before? :thinking:

Reply:
No, I don't think we were running lavamoat before in `yarn start:test`, but with the MV3 change, something broke, and this had to be done to fix it.  Specifically `--apply-lavamoat=false` and not `--snow=false`.